app/controllers/provider/admin/backend_apis/metrics_controller.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+class Provider::Admin::BackendApis::MetricsController < Provider::Admin::BackendApis::BaseController
+  before_action :find_metric, except: %i[new create index]
+  before_action :build_metric, only: %i[new create]
+
+  activate_menu :backend_api, :methods_metrics
+  sublayout 'api/service'
+
+  def create
+    # TODO
+  end
+
+  def index
+    @metrics = @backend_api.metrics.top_level.includes(:proxy_rules)
```

Comment:
```suggestion
    @metrics = @backend_api.top_level_metrics.includes(:proxy_rules)
```

Method added in this same PR in https://github.com/3scale/porta/pull/1008/files#diff-fa56de9974347a6267b1a1940abc1eb5R42